tactics/class_tactics.ml

```diff
@@ -1341,7 +1341,7 @@ module Search = struct
                                             | Some i -> str ", with depth limit " ++ int i));
     tac
 
-  let run_on_evars p evm tac =
+  let run_on_evars p evm tac env =
```

Comment:
Please follow the convention that we take `env sigma` as the first arguments of functions.